src/requestHandler.ts

```diff
@@ -0,0 +1,94 @@
+import LaunchDarkly, { LDClient } from "launchdarkly-node-server-sdk"
+import { Application, Request, Response } from "express"
+
+import { LDUser, LDData } from "./types"
+
+const createLDClient = async (sdkKey): Promise<LDClient> => {
+  const ldClient = LaunchDarkly.init(sdkKey)
+  return ldClient.waitForInitialization()
+}
+
+const getLDClient = async (
+  app: Application,
+  sdkKey: string
+): Promise<LDClient> => {
+  const existingLDClient = app.get("LDClient")
+  if (existingLDClient) {
+    return existingLDClient
+  }
+
+  const newLDClient = await createLDClient(sdkKey)
+  app.set("LDClient", newLDClient)
+  return newLDClient
+}
+
+const matchesBotUserAgent = (headers) => {
+  const botList = [
```

Comment:
Id still go with a 3rd party library for this

Reply:
yeah, we can do that in a separate PR. the experiment seems to be a success so far